tuple_practice.py

```diff
@@ -0,0 +1,16 @@
+# 1. Create the list ['a', 'b', 'c'], then create a tuple from that list
+lst1 = list('abc')
```

Comment:
do we need this type conversion here?

Reply:
the way of creating list isn't mentioned.